source/tutorials/pnp-testsuites/pnp-testsuites.rst

```diff
@@ -0,0 +1,545 @@
+.. _pnp-testsuite:
+
+|C| Power and Performance Test Suite
+####################################
+
+This guide describes the workloads included in the |C| Power and Performance (PnP)
+test suite and provides details for running the tests.
+
+.. contents::
+   :local:
+   :depth: 1
+
+Introduction
+============
+
+|C| supports deployment models of bare metal, virtual machine, and container.
+|C| can be run as the only Google* Android instance on the platform, either on a physical
+hardware device or a host OS using virtualization or containerization technology.
+Or, it can be run simultaneously with other |C| instances in a virtual machine or container
+in the workload consolidation scenario.
+
+Due to the different types of deployment models, there are two supported modes
+for measuring performance and power on |C|:
+
+* **Single-OS mode**: only one |C| instance is launched. When deployed on a host OS,
+  no concurrent host OS workloads are issued, except the required device emulation
+  loads.
+
+* **Multi-OS mode**: multiple |C| instances are launched and running identical workloads.
+  No loads other than the required device emulation tasks are issued in the host OS.
+
+The |C| power and performance measurement workloads cover the general purpose
+usage scenario for Edge devices, as well as scenarios for visual collaboration and portable devices.
+The full suite of test workloads can be used for Single OS tests, and part of the workloads
+suite can be modified for Multi-OS tests.
+While the Single-OS test items are designed to explore both hardware capability and
+the end user experience, the Multi-OS test items are focused on the end user experience of
+each instance when multiple OSes are running concurrently. In the Multi-OS profile, hardware
+capabilities are shared among multiple instances and the acceptable user experience
+of each instance is the key indicator to deliver. Refer to the following table for
+more information about the test workloads.
+
+.. list-table:: Celadon Power and Performance Test Suite
+    :header-rows: 1
+
+    * - Category
+      - :abbr:`Celadon PnP Test Suite (eliminate disadvantage & build advantage of intel architecture on android ecosystem against competitor)`
```

Comment:
I am not sure how this text is supposed to render in your final output. It seems too long for a column heading and perhaps it should be pulled out into the paragraph before the table. 
For legal/branding reasons, please change "intel architecture" to "Intel® architecture" and "android" to "Android*"